applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -60,6 +60,7 @@ def ExecuteFinalizeSolutionStep(self):
 
         absolute_tolerance = 1e-9
         CPFApp.PotentialFlowUtilities.CheckIfWakeConditionsAreFulfilled2D(self.wake_sub_model_part, absolute_tolerance, self.echo_level)
+        CPFApp.PotentialFlowUtilities.ComputePotentialJump2D(self.wake_sub_model_part)
```

Comment:
Could you please also add it to the wake process in 3d.py

Reply:
Sure, I thought it was better to wait for #8842, but I can add it here.